lib/scala/docs-generator/src/bench/scala/org/enso/docs/generator/DocsGeneratorBench.scala

```diff
@@ -0,0 +1,56 @@
+package org.enso.docs.generator
+
+import org.enso.docs.generator.DocsGenerator
+import org.scalameter.api._
+import org.scalameter.execution.LocalExecutor
+import org.scalameter.picklers.Implicits._
+import scala.annotation.nowarn
+import scala.math.pow
+
+object DocsGeneratorBench extends Bench.OfflineRegressionReport {
+
+  override def executor = new LocalExecutor(warmer, aggregator, measurer)
+
+  val range = 0
+  @nowarn("cat=w-flag-numeric-widen")
```

Comment:
Can't we get rid of this warning by inserting an explicit `.toDouble` or something similar?